app/controllers/weets.js

```diff
@@ -0,0 +1,14 @@
+const service = require('../services/weets');
+
+const generateWeet = async (_, res) => {
+  try {
+    const { data } = await service.fetchWeet();
+    res.status(200).send(data);
+  } catch (e) {
+    res.status(400).send(e.message);
+  }
+};
+
+module.exports = {
+  generateWeet
+};
```

Comment:
This is totally fine, but this card does not require the endpoint to fetch the `weets` for now. There are some things to comment here, but we'll discuss them in a future pr.